docs/superpowers/plans/2026-07-04-setup-reddit-command.md

```
@@ -0,0 +1,438 @@
+In `README.md`, section "Enable the Reddit source (optional)", replace:
+
+```markdown
+1. Create a **script** app at <https://www.reddit.com/prefs/apps> → note the **client id** (under the app name) and **secret**.
+2. Export them before running:
+
+```bash
+export OPENINTEL_REDDIT_CLIENT_ID=your_client_id
+export OPENINTEL_REDDIT_CLIENT_SECRET=your_secret
+openintel analyze AAPL --enable-reddit
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)